src/core/rendering.js

```diff
@@ -12,6 +12,7 @@ require('./p5.Graphics');
 require('./p5.Renderer2D');
 require('../webgl/p5.RendererGL');
 var defaultId = 'defaultCanvas0'; // this gets set again in createCanvas
+var defaultClass = 'p5Canvas';
```

Comment:
maybe `'p5-canvas'` would be me idiomatic for a css class. Plus this avoid any [confusion](https://stackoverflow.com/questions/12533926/are-class-names-in-css-selectors-case-sensitive)
 about case sensitive / insensitive.

Reply:
Yeah, I personally prefer `p5-canvas` as well, but as the default ID uses camel case I just thought maybe keep the casing consistent here. What do you think?

Reply:
I am not super confident about this.

I think it is better to align with the standard used elsewhere (ex [leaflet](https://www.phpclasses.org/browse/file/142068.html), [bootstrap](https://www.w3schools.com/bootstrap/bootstrap_ref_all_classes.asp), ...)

However it seems like the default ID is already [being used quite a lot](https://github.com/search?q=defaultCanvas0&type=Code&utf8=%E2%9C%93)  as a css selector. In that case you are probably right about keeping it consistent.